tests/lib/rules/no-global-regexp-flag-in-query.test.ts

```diff
@@ -0,0 +1,96 @@
+import rule, {
+  RULE_NAME,
+} from '../../../lib/rules/no-global-regexp-flag-in-query';
+import { createRuleTester } from '../test-utils';
+
+const ruleTester = createRuleTester();
+
+ruleTester.run(RULE_NAME, rule, {
+  valid: [
+    `
+        import { screen } from '@testing-library/dom'
+        screen.getByText(/hello/)
+    `,
+    `
+        import { screen } from '@testing-library/dom'
+        screen.getByText(/hello/i)
+    `,
+    `
+        import { screen } from '@testing-library/dom'
+        screen.getByText('hello')
+    `,
+
+    `
+        import { screen } from '@testing-library/dom'
+        screen.findByRole('button', {name: /hello/})
+    `,
+    `
+        import { screen } from '@testing-library/dom'
+        screen.findByRole('button', {name: /hello/i})
+    `,
+    `
+        import { screen } from '@testing-library/dom'
+        screen.findByRole('button', {name: 'hello'})
+    `,
+    `
+        const utils = render(<Component/>)
+        utils.findByRole('button', {name: /hello/i})
+    `,
+    `
+        const {queryAllByPlaceholderText} = render(<Component/>)
+        queryAllByPlaceholderText(/hello/i)
+    `,
+  ],
+  invalid: [
+    {
+      code: `
+        import { screen } from '@testing-library/dom'
+        screen.getByText(/hello/g)`,
+      errors: [
+        {
+          messageId: 'noGlobalRegExpFlagInQuery',
```

Comment:
Could you check in which row and column the error is displayed? For all invalid test cases.

Reply:
It seems this one doesn't have the line and column yet?